core/templates/pages/about-page/about-page.component.spec.ts

```diff
@@ -74,7 +74,11 @@ describe('About Page', () => {
 
     expect(siteAnalyticsService.registerCreateLessonButtonEvent)
       .toHaveBeenCalledWith();
-    expect(component.windowRef.nativeWindow.location.href).toBe(
+    // The bracket notation has been used as an escape hatch to
+    // access the private property 'windowRef'.
+    // see https://www.typescriptlang.org/play#example/private-class-fields
+    // eslint-disable-next-line dot-notation
+    expect(component['windowRef'].nativeWindow.location.href).toBe(
```

Comment:
Have tried this creating a MockWindowRef and injecting the `WindowRef` itself ?

Reply:
done